Doc/library/typing.rst

```diff
@@ -1584,16 +1584,31 @@ These are not used in annotations. They are building blocks for creating generic
 
       assert isinstance(open('/some/file'), Closable)
 
+      @runtime_checkable
```

Comment:
I think many users aren't too familiar with `OSError`'s constructor / we can't use kwargs and it's unclear what the second argument is. Maybe we could use `int` and `HasNumerator` instead?

(I know it's not writeable, but `OSError`'s errno also is maybe not writeable? Writing to errno doesn't affect repr)

Reply:
Perhaps unsurprisingly, not too many good cases of writeable attributes in stdlib. Maybe Thread.name? It's explicitly documented as something you can write to.

Reply:
That works! I'll make the change tomorrow (it's late here)